base/settings/tests/testthat/test.clean.settings.R

```diff
@@ -0,0 +1,35 @@
+PEcAn.logger::logger.setQuitOnSevere(FALSE)
+on.exit(PEcAn.logger::logger.setQuitOnSevere(TRUE))
+
+test_that("`test.clean.settings` works correctly for invalid and correct inputs", {
+
+  # Error if input file is NULL or does not exist
+  expect_error(
+    clean.settings(inputfile = NULL), 
+    "Could not find input file."
+  )
+  expect_error(
+    clean.settings(inputfile = "nonexistent.xml"), 
+    "Could not find input file."
+  )
+
+  # Works if correct input file provided
+  t <- withr::with_tempfile("tf", {
```

Comment:
A more informative name would help here too. Maybe `cleaned_settings`?

Reply:
Or `res`, which is still short but is a familiar name for this kind of "here, hold this output for a sec" variable.